modules/acme_bucket/main.tf

```diff
@@ -4,17 +4,10 @@ data "aws_s3_bucket" "logging_bucket" {
 
 resource "aws_s3_bucket" "acme_bucket" {
   bucket = var.bucket_name
-
+  acl    = "public-read"
   logging {
     target_bucket = data.aws_s3_bucket.logging_bucket.id
     target_prefix = format("%s/logs/", var.bucket_name)
   }
 
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
-    }
-  }
 }
```

Comment:
tfsec check aws-s3-specify-public-access-block failed. 

Resource module.acme_finance_bucket:aws_s3_bucket.acme_bucket has no associated aws_s3_bucket_public_access_block.

For more information, see:

- https://aquasecurity.github.io/tfsec/latest/checks/aws/s3/specify-public-access-block
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_public_access_block#bucket
- https://docs.aws.amazon.com/AmazonS3/latest/userguide/access-control-block-public-access.html